Implement watch mode

tmair opened the pull request.

ghost commented:
There were the following issues with this Pull Request

* Commit: c5165b320fd3874c396832dcfef35038a594dbac
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

tmair committed:
feat: Add watch mode for main process files

Add a typescript watcher to watch for changes to the main process files. Start the electron process
only after angular and the main process sources are compiled for the first time

fix #1
